ui/src/app/views/workspace/component.html

```diff
@@ -5,7 +5,7 @@
     <h3>CDR Version</h3>
     <div class="indented">
       <!-- TODO: Add CDR Version Info - blrubenstein -->
-      <p>CDR version info goes here.</p>
+      <p id="cdr-text">CDR version info goes here.</p>
```

Comment:
I tested out whether IDs could guarantee uniqueness (i.e., would error if there were multiple of the same) and they do not. Because of that, I moved the ids to class names. (And in some cases made them more distinct.)